ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -36,13 +36,16 @@ import           Data.Map.Strict (Map)
 import           Data.Maybe (isJust, mapMaybe)
 import           Data.Proxy
 import qualified Data.Text as Text
+import           Data.Time.Clock (UTCTime)
```

Comment:
I don't think we still need changes in this file?

Reply:
Yes indeed, that was an artefact from a previous merge + later rebase. I've removed all those from the PR.